src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -226,6 +226,26 @@ namespace AirflowNetwork {
     int constexpr AirflowNetworkControlMultiADS(5); // Perform distribution system during system on time
                                                     // and multizone calculations during off time
 
+    void generic_crack(Real64 &coef,             // Flow coefficient
+                       Real64 const expn,        // Flow exponent
+                       bool const LFLAG,         // Initialization flag.If = 1, use laminar relationship
+                       Real64 const PDROP,       // Total pressure drop across a component (P1 - P2) [Pa]
+                       const AirState &propN,    // Node 1 properties
+                       const AirState &propM,    // Node 2 properties
+                       std::array<Real64, 2> &F, // Airflow through the component [kg/s]
+                       std::array<Real64, 2> &DF // Partial derivative:  DF/DP
+    );
```

Comment:
Is this intentionally named in camel_case while GenericDuct is PascalCase?

Reply:
Yes, I haven't worked on GenericDuct and I've left it in the original form to remind me of that. When I do get to it, maybe later this year, that'll mean simulation diffs so I want to leave it as is for now.